src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -71,6 +71,9 @@ const propTypes = {
 
     // Whether we have the necessary report data to load the sidebar
     initialReportDataLoaded: PropTypes.bool,
+
+    // Syncing App Data
```

Comment:
This is picky but would you mind changing this comment to match the other comments something like:
```suggestion
    // Whether we are syncing app data
```